test/show.jl

```diff
@@ -64,3 +63,46 @@ end
      t0: 0
      b: 0"""
 end
+
+@testset "KeplerianOrbit" begin
+    orbit = KeplerianOrbit(
+        ρ_star = 1.0,
+        R_star = cbrt(3.0/(4.0*π)),
+        period = 2.0,
+        ecc = 0.0,
+        t_0 = 0.0,
+        incl = π / 2.0,
+        Ω = 0.0,
+        ω = 0.0,
+    )
+
+    @test sprint(show, "text/plain", orbit) === """
+    Keplerian Orbit
+     P:      2.0000 d
+     t₀:     0.0000 d
+     tₚ:     -0.5000 d
+     t_ref:  -0.5000 d
+     τ:      nothing
+     a:      6.6802 R⊙
+     aₚ:     -6.6802 R⊙
+     aₛ:     0.0000 R⊙
+     Rₚ:     0.0000 R⊙
+     Rₛ:     0.6204 R⊙
+     ρₚ:     NaN M⊙/R⊙³
+     ρₛ:     1.0000 M⊙/R⊙³
+     r:      0.0000
+     aRₛ:    10.7685
+     b:      0.0000
+     ecc:    0.0000
+     cos(i): 0.0000
+     sin(i): 1.0000
+     cos(ω): 1.0000
+     sin(ω): 0.0000
+     cos(Ω): 1.0000
+     sin(Ω): 0.0000
+     i:      1.5708 rad
+     ω:      0.0000 rad
+     Ω:      0.0000 rad
+     Mₚ:     0.0000 M⊙
+     Mₛ:     1.0000 M⊙"""
+end
```

Comment:
Can you add a `repr` test for a Unitful.jl constructed orbit?

Reply:
f0e478a f7ed0c4

I also added the preferred units to the test harness, similarly to [here](https://github.com/JuliaAstro/Transits.jl/pull/23#commitcomment-58931035)